docs/design/datacontracts/GCHandle.md

```diff
@@ -0,0 +1,23 @@
+# Contract GCHandle
+
+This contract allows decoding and reading of GCHandles. This will also include handle enumeration in the future
```

Comment:
> This will also include handle enumeration in the future

I would think that GCHandle enumeration is going to be separate from GCHandle dereferencing.

GCHandle is very simple and it is needed for minidump analysis.

GCHandle enumeration is much more involved and it is not needed for minidump analysis.

Reply:
Fair. I can easily see stopping at supporting only individual `GCHandle` here, and defining the enumerators separately. We may want to provide the ability to determine what kind of G`CHandle`, and possibly the various other characteristics of a non-boring `GCHandle`.

Reply:
The split that I'm guessing would work well is:
- A contract for Object that includes the facts that OBJECTREF can be treated as Object* and OBJECTHANDLE can be treated as Object**
- A GCHandle contract that has all the other details for other handle state and handle enumeration.

Fwiw I don't recall that ICorDebug or SOS has any exposure of GCHandle information that isn't combined with enumerating them first.